### PR TITLE
Applications pt 1

### PR DESCRIPTION
**Pull request by bswags**
Able to apply with projects! A follow-up will plumb this data into the applications page so that you can't re-apply

### CODE REVIEW

File: src/components/application/ApplicationSubmitted.tsx [hunk 1]
```diff
@@ -47,14 +50,17 @@ export const ApplicationSubmitted = ({ className }: { className?: string }) => {
...
-        </h2>
-        <p className="text-center">
-          You&apos;ll receive an email at{" "}
-          <span className="font-medium text-muted-foreground">
-            shaun@optimism.io
-          </span>{" "}
+        <h2 className="text-center">Application Submitted!</h2>
+        <p className="text-center text-lg font-semibold max-w-lg">
+          You&apos;ll receive an email{" "}
+          {email ? (
```
Comment (tinaszheng) [inline src/components/application/ApplicationSubmitted.tsx:56]: oooh what if they don't have an email? maybe we can pop up the email modal

File: src/lib/actions/contracts.ts [hunk 2]
```diff
@@ -31,6 +32,11 @@ export const verifyContract = async ({
     }
   }
 
+  const isInvalid = await verifyMembership(projectId, session.user.farcasterId)
```
Comment (bswags) [inline src/lib/actions/contracts.ts:35]: this was missing before oops

**Review by tinaszheng — APPROVED** [review]
nice

File: src/components/application/ApplicationSubmitted.tsx (hunk 1 above)
Comment (bswags) [reply src/components/application/ApplicationSubmitted.tsx:56]: i believe we're supposed to pop it on the previous step in the dashboard, but also you could just link straight here? hmmm good point let's add this to the follow-ups list

**Pull request merged**